scripts/kubestellar

```diff
@@ -320,6 +348,17 @@ if [[ ! -d $log_folder ]]; then
     mkdir -p $log_folder
 fi
 
+space-manager -kubeconfig $KUBECONFIG -context "root:"$space_mgt_name -v=2 >& $log_folder/space-manager-log.txt &
```

Comment:
Why is `-kubeconfig $KUBECONFIG` needed here?  The `space-manager` command should be sensitive to the `KUBECONFIG` envar like any other command.

Reply:
1. It is not necessary. Can be removed.

2. Will change to append

Reply:
Done.